test/unit_tests/test_braket_device.py

```diff
@@ -1314,6 +1313,8 @@ def test_execute_some_samples(mock_run):
     mock_run.return_value = task
     dev = _aws_device(wires=3)
 
+    print(type(qml.Hadamard(0) @ qml.Identity(1)))
```

Comment:
nit: Do we need this print, should this type check be asserted instead?

Reply:
Ah, we don't need this at all! 

A previous test (now removed) switched the default behaviour of the `@` operator to the legacy behaviour and forgot to switch it back, so this test was actually running with the legacy code instead of the current code and creating an incorrect operator type. I just added this when I was debugging what was going on there. 

But it's fixed now, and the legacy code doesn't exist anymore, so I'll just delete it completely.